smoke-tests/matrix/settings.gradle

```diff
@@ -1 +1,3 @@
 rootProject.name = 'matrix'
+
+include ':servlet-jakarta'
```

Comment:
I think would be nice to rename to `servlet-5.0` and move the existing `3.0` web app into a sibling `servlet-3.0`, so that it's more clear they are copies targeting different servlet versions (hopefully less likely we will forget to update one when we update the other)

Reply:
Done, moved the 3.0 web app to `servlet-3.0` module and renamed `servlet-jakarta` to `servlet-5.0`.